doc/poster/pasc22.html

```diff
@@ -0,0 +1,179 @@
+      <p>Its simplified data-management does not come at a loss of any of HDF5's fledibility and
+      advanced features and tunable parameters are exposed through a simple interface:</p>
+      <div class="card w-75"> 
+      <div class="code card-body">
+     <pre class="font-monospace"><div class="example" id="example2"></div></pre>
+    </div>
+    </div>
+    </div>
+    <div class="container">
+      <p>HighFive supports scalar and compound types as well as boost, Eigen and XTensor array
+      types:</p>
+      <div class="card w-75">
+        <div class="card-body">
+          <ul class="nav nav-tabs" id="highfive-type-support" role="tablist">
+            <li class="nav-item" role="presentation">
+              <button class="nav-link active" id="compound-tab" data-bs-toggle="tab"
+                                                                data-bs-target="#compound" type="button" role="tab" aria-controls="compound"
+                                                                                                                    aria-selected="true">Compound types</button>
+            </li>
+            <li class="nav-item" role="presentation">
+              <button class="nav-link" id="boost-tab" data-bs-toggle="tab" data-bs-target="#boost"
+                                                                           type="button" role="tab" aria-controls="boost" aria-selected="false">Boost</button>
+            </li>
+            <li class="nav-item" role="presentation">
+              <button class="nav-link" id="eigen-tab" data-bs-toggle="tab" data-bs-target="#eigen"
+                                                                           type="button" role="tab" aria-controls="eigen" aria-selected="false">Eigen</button>
+            </li>
+          </ul>
+          <div class="tab-content" id="highfive-type-support-content">
+            <div class="tab-pane fade show active" id="compound" role="tabpanel"
+                                                                 aria-labelledby="compound-tab">
+              <div class="code container">
+                <pre class="font-monospace"><div class="example" id="example3"></div></pre>
+              </div>
+            </div>
+            <div class="tab-pane fade" id="boost" role="tabpanel" aria-labelledby="boost-tab">
+              <div class="code container">
+                <pre class="font-monospace"><div class="example" id="example4"></div></pre>
+              </div>
+            </div>
+            <div class="tab-pane fade" id="eigen" role="tabpanel" aria-labelledby="eigen-tab">
+              <div class="code container">
+                <pre class="font-monospace"><div class="example" id="example5"></div></pre>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    </div>
+    <div class="container">
+      <p>HighFive is both thread-safe and supports the HDF5 MPI backend and can be natively used in
```

Comment:
```suggestion
      <p>HighFive supports the HDF5 MPI backend and can be natively used in
```
Thread-safety is a feature of HDF5 IMO…

Reply:
I had a doubt there myself, but had taken it from the abstract.